### ISSUE TITLE
How to reproduce the CIFAR10 validation accuracy?

### ISSUE THREAD

**Issue by nameoverflow** [xiaomi-automl/FairDARTS#5]
I tried the default settings in [fairdarts/train.py ](https://github.com/xiaomi-automl/FairDARTS/blob/master/fairdarts/train.py) with FairDARTS-a model, only reaching ~97.0% accuracy on CIFAR10, though the pretrained weights could get 97.46%. Also I found this repo does not enable `cudnn.deterministic`. Could you please provide deterministic seed for reproduction, or is there any extra operations for repreducing the results?

**Comment by tianbaochou** [comment]
> I tried the default settings in [fairdarts/train.py ](https://github.com/xiaomi-automl/FairDARTS/blob/master/fairdarts/train.py) with FairDARTS-a model, only reaching ~97.0% accuracy on CIFAR10, though the pretrained weights could get 97.46%. Also I found this repo does not enable `cudnn.deterministic`. Could you please provide deterministic seed for reproduction, or is there any extra operation for reproducing the results?

We train the fairdarts-a with epoch 600, batch-size 96 and achieve about 97.18. In our experiments, we found batch-size=128 or 256 is not good.
Wthin FairDarts-a, b, c...， Our paper result is training epoch=1600 and batch-size=96 for convergence enough.
